custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js

```diff
@@ -129,49 +128,67 @@ function ServiceDeliveryDashboardController($scope, $http, $location, $routePara
         return renderHeaderTooltip('Growth Monitoring', 'Of the total children between <b>3-5 years</b> of age and enrolled for Anganwadi services, the percentage of children who were weighed in the current month.<br><br><b>Growth Monitoring is done only for children till 5 years of age.</b>');
     }
 
-    function renderPercentageAndPartials(percentage, nominator, denominator) {
-        if (denominator === vm.dataNotEntered) { return vm.dataNotEntered; }
-        if (percentage === vm.dataNotEntered) {
-            if (nominator === 0 && denominator === 0) {
-                return '<div><span>100 %<br>(' + nominator + ' / ' + denominator + ')</span></div>';
+    function isZeroNullUnassignedOrDataNotEntered(value) {
+        return value === 0 || value === null || value === void(0) || value === vm.dataNotEntered;
+    }
+
+    function renderPercentageAndPartials(percentage, nominator, denominator, indicator) {
+        if (haveAccessToFeatures && isZeroNullUnassignedOrDataNotEntered(denominator)) {
```

Comment:
where does `haveAccessToFeatures` come from? its not passed into the function? Also is this text intended to be feature flagged?

Reply:
hey it is passed in line 234 below if you see. and yes its asked to be in feature flag first.

Reply:
I see